### PR TITLE
Update FATTestIDSWithSSL to use Unboundid

### LINKED ISSUE #12242: Unexpected Exception found in FATTestIDSWithSSL
getGroupSecurityNameWithInvalidGroup:java.lang.Exception: 2020-05-01-11:29:12:759 Unexpected exception expected<com.ibm.ws.security.registry.EntryNotFoundException> but was<com.ibm.ws.security.registry.RegistryException>
at componenttest.custom.junit.runner.FATRunner$1.evaluate(FATRunner.java:193)
at componenttest.custom.junit.runner.FATRunner$2.evaluate(FATRunner.java:314)
at componenttest.custom.junit.runner.FATRunner.run(FATRunner.java:167)

### PR DESCRIPTION
**Pull request by agiridharan**
fixes #12242

### CODE REVIEW

**Comment by agiridharan** [comment]
#build

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat/fat/src/com/ibm/ws/security/wim/adapter/ldap/fat/FATTestIDSwithSSL.java [hunk 1]
```diff
@@ -45,16 +50,48 @@
     private static final Class<?> c = FATTestIDSwithSSL.class;
     private static UserRegistryServletConnection servlet;
 
+    /** Test rule for testing for expected exceptions. */
+    @Rule
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.adapter.ldap_fat/fat/src/com/ibm/ws/security/wim/adapter/ldap/fat/FATTestIDSwithSSL.java:54]: Not sure why this is here. I don't see you use it anywhere.

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat/fat/src/com/ibm/ws/security/wim/adapter/ldap/fat/FATTestIDSwithSSL.java [hunk 2]
```diff
@@ -23,10 +23,15 @@
 
 import org.junit.AfterClass;
 import org.junit.BeforeClass;
+import org.junit.Rule;
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.adapter.ldap_fat/fat/src/com/ibm/ws/security/wim/adapter/ldap/fat/FATTestIDSwithSSL.java:26]: Update the copyright above.

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.registry.ldap.fat.ids.ssl/server.xml [hunk 3]
```diff
@@ -44,7 +44,7 @@
 	<sslDefault sslRef="DefaultSSLSettings" />
 	<ssl id="LDAPSSLSettings" keyStoreRef="LDAPKeyStore" trustStoreRef="LDAPTrustStore" />
 	<keyStore id="LDAPKeyStore" location="${server.config.dir}/LdapSSLKeyStore.jks" type="JKS" password="{xor}CDo9Hgw=" />
-	<keyStore id="LDAPTrustStore" location="${server.config.dir}/LdapSSLTrustStore.jks" type="JKS" password="{xor}CDo9Hgw=" />
+	<keyStore id="LDAPTrustStore" location="${server.config.dir}/truststore.p12" type="PKCS12" password="LDAPpassword" />
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.registry.ldap.fat.ids.ssl/server.xml:47]: The failover servers should be removed.

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat/fat/src/com/ibm/ws/security/wim/adapter/ldap/fat/FATTestIDSwithSSL.java [hunk 4]
```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2011 IBM Corporation and others.
+ * Copyright (c) 2020 IBM Corporation and others.
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.adapter.ldap_fat/fat/src/com/ibm/ws/security/wim/adapter/ldap/fat/FATTestIDSwithSSL.java:2]:
```suggestion
 * Copyright (c) 2011, 2020 IBM Corporation and others.
```

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat/fat/src/com/ibm/ws/security/wim/adapter/ldap/fat/FATTestIDSwithSSL.java [hunk 5]
```diff
@@ -45,16 +49,48 @@
     private static final Class<?> c = FATTestIDSwithSSL.class;
     private static UserRegistryServletConnection servlet;
 
+    /** Test rule for testing for expected exceptions. */
+
+    public ExpectedException expectedException = ExpectedException.none();
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.adapter.ldap_fat/fat/src/com/ibm/ws/security/wim/adapter/ldap/fat/FATTestIDSwithSSL.java:54]: I don't think you use expectedException.

```suggestion
```

File: dev/com.ibm.ws.security.wim.adapter.ldap_fat/fat/src/com/ibm/ws/security/wim/adapter/ldap/fat/FATTestIDSwithSSL.java [hunk 6]
```diff
@@ -45,16 +49,48 @@
     private static final Class<?> c = FATTestIDSwithSSL.class;
     private static UserRegistryServletConnection servlet;
 
+    /** Test rule for testing for expected exceptions. */
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.wim.adapter.ldap_fat/fat/src/com/ibm/ws/security/wim/adapter/ldap/fat/FATTestIDSwithSSL.java:52]:
```suggestion
```

**Comment by agiridharan** [comment]
#build

**Comment by jvanhill** [comment]
#libby

**Comment by agiridharan** [comment]
#run-libby-bot

**Pull request merged**